Dart/src/com/jetbrains/lang/dart/ide/completion/DartServerCompletionContributor.java

```diff
@@ -605,7 +605,8 @@ private static CompletionSuggestion createCompletionSuggestionFromAvailableSugge
       null,
       null,
       null,
-      null);
+      null,
+      0);
```

Comment:
According to the `libraryUriToImportIndex` field docs, I think it's better to pass `null` here instead of `0`. Would you agree?

It doesn't matter in this particular case (as it's not used), just for the consistent use of the API.

Reply:
Yes, `null` is fine, but it also doesn't matter for the time being.

> May I ask how these files are generated?
> 
> I tried to run `generate_all` in Dart analyzer packages, but the output doesn't contain the relevant sources.

@asashour the consumers aren't generated, they are done manually.  The output from `generate_all` is the correct location.

Reply:
@jwren  Thanks for your reply.

I guess that the comment about the other PR https://github.com/JetBrains/intellij-plugins/pull/832#issuecomment-942475198, that all files should not be changed is not precise.

I would raise a CL for dart generation, then we come to the `intellij-plugins` later.